src/js/vre/chapter31/config/form.js

```diff
@@ -61,12 +76,39 @@ const formConfig = {
     workInformation: {
       title: 'Work Information',
       pages: {
-        veteranInformation: {
+        workInformation: {
           path: 'work-information',
           title: 'Work Information',
+          uiSchema: {
+            'view:isWorking': {
+              'ui:title': 'Are you working?',
+              'ui:widget': 'yesNo'
+            },
+            employer: _.merge(
+              {
+                'ui:title': 'Employer name'
+              },
+              hideIfNotWorking,
```

Comment:
Not sure if this is an idiomatic approach.

Reply:
What we probably want to do is use the `expandUnder` functionality, so that `employer` expands under `view:isWorking`.